git-ftp

```diff
@@ -789,11 +789,14 @@ submodule_catchup() {
 	set_submodule_args
 	url="$(git config git-ftp.url)"
 	print_info "Submodules are $GIT_SUBMODULES"
-	for submodule in "${GIT_SUBMODULES[@]}"
+	local OIFS="$IFS"
```

Comment:
If `GIT_SUBMODULES` is an array as described above, then you can restore the original version again without changing `IFS`:
```bash
for submodule in "${GIT_SUBMODULES[@]}"
```